### PR TITLE
add Color page

### PR DESCRIPTION
**Pull request by hsnice16**
This PR will add a Color page in the development branch.

UI of the Color page for different screens,
image

image

image

Please review this PR, and give your feedback.

### CODE REVIEW

**Review by SukanyaSen530 — APPROVED** [review]

**Review by Developers-codz — APPROVED** [review]
Looking Good to me!

File: app.css [hunk 1]
```diff
@@ -152,6 +154,10 @@ body {
     display: none;
   }
 
+  .component-container > .grid {
+    grid-template-columns: minmax(0, 1fr);
+  }
+
   .link-to-top {
```
Comment (manojsarna) [inline app.css:161]: when to use unset , can you elaoborate on this!!

File: css/docs.css [hunk 2]
```diff
@@ -63,7 +63,7 @@
   overflow: hidden;
 }
 
-.component-container > div {
+.component-container > div:not(.grid) {
   padding: 2rem 2rem 0;
 }
```
Comment (manojsarna) [inline css/docs.css:68]: can you elaborate on this?

File: pages/components/toast.html [hunk 3]
```diff
@@ -415,6 +419,11 @@ <h3 class="my-1p5">On this Page</h3>
...
           </a>
         </div>
+        <div class="ml-auto">
+          <a class="link" href="tooltip.html">
+            Tooltip <i class="fas fa-hand-point-right"></i>
+          </a>
+        </div>
       </footer>
 
       <!-- Footer Docs Ends -->
```
Comment (manojsarna) [inline pages/components/toast.html]: this is superb , using superb coding practices !!

File: pages/components/tooltip.html [hunk 4]
```diff
@@ -0,0 +1,267 @@
...
+              <button class="bg-red-50 card-icon-btn mx-0p5 tooltip">
+                <i class="fas fa-heart m-auto"></i>
+                <span class="tooltip-right tooltip-text">
+                  remove from wishlist
+                </span>
+              </button>
+            </div>
+
+            <pre class="pre-code">
+              <code class="language-html">
```
Comment (manojsarna) [inline pages/components/tooltip.html]: this is superb

**Review by manojsarna — APPROVED** [review]
Overall Code Looks great

Good Naming Conventions

File: css/docs.css (hunk 2 above)
Comment (hsnice16) [reply css/docs.css:68]: yes, sure. here I am selecting all `div` that do not have `grid` class, and are a direct child of element having class `component-container`.

File: app.css (hunk 1 above)
Comment (hsnice16) [reply app.css:161]: yeah, sure. Actually initially `link-to-top` has a `display: none` class, which I have set in `docs.css`. And when the screen width is coming down to `63em`, I am unsetting that display. Basically whatever it has default `display` value, just saying to apply that.

And when the screen width comes down to `44em`, I am again setting it to `display: none`. It is for responsiveness. 

If you want to read more about this, you can refer this, [unset mdn](https://developer.mozilla.org/en-US/docs/Web/CSS/unset)

**Comment by hsnice16** [comment]
Thank you for reviewing this.

**Pull request merged**